lib/Int64.fram

```diff
@@ -0,0 +1,71 @@
+pub method equal {self : Int64} = (extern dbl_eqInt64  : Int64 -> Int64 -> Bool) self
+pub method neq   {self : Int64} = (extern dbl_neqInt64 : Int64 -> Int64 -> Bool) self
+pub method gt    {self : Int64} = (extern dbl_gtInt64  : Int64 -> Int64 -> Bool) self
+pub method lt    {self : Int64} = (extern dbl_ltInt64  : Int64 -> Int64 -> Bool) self
+pub method ge    {self : Int64} = (extern dbl_geInt64  : Int64 -> Int64 -> Bool) self
+pub method le    {self : Int64} = (extern dbl_leInt64  : Int64 -> Int64 -> Bool) self
+
+pub method toString = (extern dbl_int64ToString : Int64 -> String) self
+
+pub method neg {self : Int64} = (extern dbl_negInt64 : Int64 -> Int64) self
+pub method add {self : Int64} = (extern dbl_addInt64 : Int64 -> Int64 -> Int64) self
+pub method sub {self : Int64} = (extern dbl_subInt64 : Int64 -> Int64 -> Int64) self
+pub method mul {self : Int64} = (extern dbl_mulInt64 : Int64 -> Int64 -> Int64) self
+
+pub let zero = 0L
+pub let one  = 1L
+pub let minus_one = (1L).neg
+pub method succ {self : Int64} = self + one
+pub method pred {self : Int64} = self - one
```

Comment:
What is the motivation behind these two methods? Why the programmer cannot just write `x + 1L`?

Reply:
as above